src/boot/pprint.ml

```diff
@@ -1,244 +1,492 @@
-
-(*
-   Miking is licensed under the MIT license.
-   Copyright (C) David Broman. See file LICENSE.txt
-*)
+(** Pretty-printing for data types in Miking.
+ *
+ *  The main functions that should be used outside of this module is
```

Comment:
Maybe we should consider making a `.mli`? Maybe overkill for `boot`?

Reply:
I thought about this as well